### DISCUSSION TITLE
CONTRIBUTING.md points to missing target file

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-tycho/tycho#2891] [Q&A]
CONTRIBUTING.md tells me to use tycho-bundles-target/tycho-bundles-target.target but I cannot find this file using `find . -name "tycho-bundles-target.target"`. Also the mentioned folder seems missing.

```
For Tycho only: Configure the target platform: Open the file tycho-bundles-target/tycho-bundles-target.target and click on Set as Target Platform in the upper right corner of the target definition editor.
```

**Comment by laeubi** [comment]
Tycho no longer requires a target file as everything are plain maven-projects now.

**Reply by vogella** [reply]
I plan to provide a PR to update the CONTRIBUTING.md and also the Oomph description (cc @merks) (which also mentions this target file)... If you do the update before me, I will not be sad...
